templates/ng1/baseClient.ejs

```diff
@@ -102,7 +102,7 @@ abstract class BaseService {
     var data = response.data;
     var status = response.status;
 
-    if (status === 200 || status === 204) {
+    if (status >= 200 && status <= 299) {
```

Comment:
Can we change it forr all templates, not only for ng1?

Reply:
Other clients do not require such changes.